server/controllers/stock/core.js

```diff
@@ -35,9 +35,27 @@ const flux = {
   INVENTORY_ADJUSTMENT : 15,
 };
 
+const fluxLabel = {};
```

Comment:
To save space, you could also write: 
```js
const fluxLabel = Object.entries(flux)
  .reduce((map, [key, value]) => { map[value] = key; return map; }, {});
```
To accomplish the same thing.  It would also prevent typos.